package.json

```diff
@@ -10,7 +10,10 @@
     "serve:server": "npm run build:server && nodemon dist/app.js",
     "serve:client": "cd client && npm run dev",
     "dev": "concurrently \"npm run serve:server\" \"npm run serve:client\"",
-    "build:server": "rm -rf ./dist/ && tsc"
+    "build:server": "rm -rf ./dist/ && tsc",
+    "build:client": "cd client && npm run build",
+    "serve:prod": "npm run build:client && npm run serve:server",
```

Comment:
To make code simpler and more readable I would recommend to sort entries alphabetically